modules/ROOT/pages/functions/temporal/index.adoc

```diff
@@ -168,6 +168,7 @@ The following table lists the different sub-functions for specifying the clock t
 
 A temporal instant value can be created by truncating another temporal instant value at the nearest preceding point in time at a specified component boundary (namely, a _truncation unit_).
 A temporal instant value created in this way will have all components which are less significant than the specified truncation unit set to their default values.
+// what about these?
```

Comment:
line 170 (and many more occurrences in this file) also use the wording "less significant", but from what i see it refers to first order components and the hierarchy among them (a month being more significant than, say, a day). still thought it was best to bring it up @JPryce-Aklundh

Reply:
Could we not just say "smaller"/"greater" - e.g "have all components which are smaller than the specified truncation..." - I think most readers will grasp that a day is "smaller" than a "month" and so on?

Reply:
updated